decidim-elections/app/models/decidim/elections/question.rb

```diff
@@ -23,8 +23,23 @@ def valid_max_selection?
         max_selections <= answers.count
       end
 
+      def blank_votes
+        @blank_votes ||= results.blank_question.sum(:votes_count)
```

Comment:
Out of curiosity: What is a `blank_question`?

Reply:
It refers to the count of blank answers for that question, it's associated to a question since NOTA answers have record in `Answers` table.
 
As [commented above](https://github.com/decidim/decidim/pull/7804#discussion_r609416770), I'll rename it to `blank_answer`, as it's more accurate.